bulbs/special_coverage/models.py

```diff
@@ -138,8 +154,7 @@ def get_content(self, published=True):
     def is_active(self):
         now = today_as_utc_datetime()
         if self.start_date and self.end_date:
-            if self.start_date < now and self.end_date > now:
-                return True
+            return self.start_date <= now < self.end_date
```

Comment:
We've been following a "start <= now < end" elsewhere throughout codebase (vs "start < now < end")